examples/dynamoathenamigration/migration_test.go

```diff
@@ -419,6 +448,13 @@ func TestMigrationCheckpoint(t *testing.T) {
 	})
 }
 
+func generateLargeEventLine() string {
+	// generate event close to 400KB which is max of dynamoDB to test if
+	// it can be processed
+	lineFmt := `{ "Item": { "EventIndex": { "N": "2147483647" }, "SessionID": { "S": "4298bd54-a747-4d53-b850-83ba17caae5a" }, "CreatedAtDate": { "S": "2023-05-22" }, "FieldsMap": { "M": { "cluster_name": { "S": "%s" }, "uid": { "S": "%s" }, "code": { "S": "T2005I" }, "ei": { "N": "2147483647" }, "time": { "S": "2023-05-22T12:12:21.966Z" }, "event": { "S": "session.upload" }, "sid": { "S": "4298bd54-a747-4d53-b850-83ba17caae5a" } } }, "EventType": { "S": "session.upload" }, "EventNamespace": { "S": "default" }, "CreatedAt": { "N": "1684757541" } } }`
+	return fmt.Sprintf(lineFmt, strings.Repeat("a", 1024*400 /* 400 KB */), uuid.NewString())
+}
```

Comment:
Indent for readability? (I just passed it through `jq`.)

```suggestion
	// Generate event close to 400KB which is max of dynamoDB to test if
	// it can be processed.
	return fmt.Sprintf(
		`{
			"Item": {
				"EventIndex": {
					"N": "2147483647"
				},
				"SessionID": {
					"S": "4298bd54-a747-4d53-b850-83ba17caae5a"
				},
				"CreatedAtDate": {
					"S": "2023-05-22"
				},
				"FieldsMap": {
					"M": {
						"cluster_name": {
							"S": "%s"
						},
						"uid": {
							"S": "%s"
						},
						"code": {
							"S": "T2005I"
						},
						"ei": {
							"N": "2147483647"
						},
						"time": {
							"S": "2023-05-22T12:12:21.966Z"
						},
						"event": {
							"S": "session.upload"
						},
						"sid": {
							"S": "4298bd54-a747-4d53-b850-83ba17caae5a"
						}
					}
				},
				"EventType": {
					"S": "session.upload"
				},
				"EventNamespace": {
					"S": "default"
				},
				"CreatedAt": {
					"N": "1684757541"
				}
			}
		}`,
		strings.Repeat("a", 1024*400 /* 400 KB */), uuid.NewString())
```